src/js/vendor/docsearch.bundle.js

```diff
@@ -7,19 +7,33 @@
   link.rel = 'stylesheet'
   link.href = config.stylesheet
   document.head.appendChild(link)
-  var search = docsearch({
+  var defaultAlgoliaOptions = { hitsPerPage: parseInt(config.maxResults) || 5 }
+  var docsearchComponent = docsearch({
     appId: config.appId,
     apiKey: config.apiKey,
     indexName: config.indexName,
     inputSelector: '#search-input',
     autocompleteOptions: { hint: false, keyboardShortcuts: ['s'] },
-    algoliaOptions: { hitsPerPage: parseInt(config.maxResults) || 5 },
-  }).autocomplete
-  var autocomplete = search.autocomplete
+    algoliaOptions: defaultAlgoliaOptions,
+  })
+  var search = docsearchComponent.autocomplete
```

Comment:
Previously, `autocomplete` and `search` were equal because we were calling `.autocomplete` on `docsearch({}).autocomplete`.

I simplified the code and assigned `docsearch` to a variable named `docsearchComponent` to be able to update `algoliaOptions` attribute.

Reply:
Yep, that's fine.